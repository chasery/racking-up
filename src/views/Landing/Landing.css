.Hero {
  position: relative;
  z-index: 2;
  padding-top: 3.75rem;
  background-color: #ebebeb;
}
.Hero .Form {
  margin: 1.5rem 1.5rem -3rem 1.5rem;
  min-width: calc(100% - 6rem);
}

.AppInfo {
  position: relative;
  z-index: 1;
  flex-grow: 1;
}
.AppInfo__wrapper {
  padding: 4.5rem 1.5rem 1rem;
}

@media all and (min-width: 30rem) {
  .Hero__wrapper {
    display: flex;
    justify-content: flex-end;
  }
}

@media all and (min-width: 50rem) {
  .AppInfo__wrapper {
    padding-top: 3rem;
  }
  .Hero__wrapper,
  .AppInfo__wrapper {
    max-width: 64rem;
    align-self: center;
    margin: 0 auto;
  }
}

@media all and (min-width: 64rem) {
  .Hero .Form {
    margin-right: 0;
  }
}
