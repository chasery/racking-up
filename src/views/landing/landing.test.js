import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Landing from "./landing";

describe("Landing component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<Landing />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
