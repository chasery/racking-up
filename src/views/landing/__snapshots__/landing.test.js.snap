// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing component renders without error 1`] = `
<Fragment>
  <Header />
  <main
    role="main"
  >
    <section
      className="Hero"
    >
      <div
        className="Hero__wrapper"
      >
        <SignUpForm />
      </div>
    </section>
    <section
      className="AppInfo"
    >
      <div
        className="AppInfo__wrapper"
      >
        <h3>
          Organize clothing purchases from online retailers
        </h3>
        <p>
          Racking Up helps you organize your upcoming wardrobe purchases with a custom rack system. With a little help from our app, you can curate lists of those clothing items you fancy from any online retailer.
        </p>
        <h3>
          Prioritize your clothing racks
        </h3>
        <p>
          Trying to work within a budget or only care about clothes for the season? Racking Up allows you to prioritize racks of clothing for purchase in a way that matters to you.
        </p>
        <h3>
          Want to give it a spin?
        </h3>
        <p>
          You can use our demo account to check out our app.
        </p>
      </div>
    </section>
  </main>
</Fragment>
`;
